02week/blog/index.html

```diff
@@ -0,0 +1,125 @@
+      tempus lacus enim ac dui. Donec non enim in turpis pulvinar facilisis. Ut felis. Praesent dapibus, neque id cursus faucibus, tortor neque egestas augue, eu vulputate magna eros eu erat. Aliquam erat volutpat. Nam dui mi, tincidunt quis, accumsan
+      porttitor, facilisis luctus, metus
+    </p>
+    <hr>
+
+    <h2>My Songwriting Process</h2>
+    <h4>Wednesday, October 2, 2017</h4>
+    <h4><div class="bynick"</div>By: Nick Andrew Gonzalez</h4>
+    <div>
+      <p>Songwriting process is so different from song to song. It could start with a beat, the instruments, a vocal melody, or just simply a melody by itself. My process may seem weird or strange, but non the less.....I'm willing to let you in and share
+        :).
+      </p>
+    </div>
+    <hr>
+
+    <h2>Behind the scenes</h2>
+    <h4>Saturday, October 5, 2017</h4>
+    <h4><div class="bynick"</div>By: Nick Andrew Gonzalez</h4>
+    <div>
+      <p>Take a behind the scenes inside look of me in the studio recording my latest single. See how the song takes life into a finished product!</p><a href="https://www.youtube.com/user/NickAndrewsGuitar" target="_blank">Check out my YouTube Channel</a>
+    </div>
+    <hr>
+
+    <h2>What truly matters<i class="em em  em-heart"></i></h2>
+    <h4>Thursday, October 10, 2017</h4>
+    <h4><div class="bynick"</div>By: Nick Andrew Gonzalez</h4>
+    <div>
+      <p>Being on the road and in this business, I meet all kinds of people from all over the world. Love, compassion, pay it forward mentality goes a long way. Here I would love to share my personal stories that have and continue to happen to me. Spread
+        love, not hate. We can all take away something from this blog.</p>
+      <hr>
+
+      <h2>Who am I listenting to??</h2>
+      <h4>Saturday, October 13, 2017</h4>
+      <h4><div class="bynick"</div>By: Nick Andrew Gonzalez</h4>
+      <div>
+        <p>What to know what latest LP's I've scored?? Want to know what's in my playlists?? Who do I think is hott right now?? Get all that good info right here!</p><br>
+        <p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae
+          est. Mauris placerat eleifend leo. Quisque sit amet est et sapien ullamcorper pharetra. Vestibulum erat wisi, condimentum sed, commodo vitae, ornare sit amet, wisi. Aenean fermentum, elit eget tincidunt condimentum, eros ipsum rutrum orci, sagittis
+          tempus lacus enim ac dui. Donec non enim in turpis pulvinar facilisis. Ut felis. Praesent dapibus, neque id cursus faucibus, tortor neque egestas augue, eu vulputate magna eros eu erat. Aliquam erat volutpat. Nam dui mi, tincidunt quis, accumsan
+          porttitor, facilisis luctus, metus</p>
+        <hr>
+
+        <h2><div class ="ewert"</div>You got questions, I got your anwers</h2>
```

Comment:
Avoid space between attribute and the equal sign
No - `class ="ewert"`
Yes- `class="ewert"``